.github/ISSUE_TEMPLATE/module-checklist.md

```diff
@@ -0,0 +1,36 @@
+---
+name: New Module / Dataset
+about: "Meta issue to track the creation, updating of a new module or dataset."
+
+---
+
+# Metricbeat Module / Dataset release checklist
+
+This checklist is intended for Devs which create or update a module to make sure modules are consistent.
+
+The idea of having this in Markdown is that it can be copied 1-1 to a Github issue and be used as meta issue.
```

Comment:
No need for this line now? :slightly_smiling_face:

Reply:
agree, removed